link_checker.py

```diff
@@ -198,6 +201,37 @@ def output_summary(num_errors):
             output_write(url)
 
 
+def check_link(link, licens_name, base_url):
```

Comment:
`licens_name` should become `license_name`

Reply:
Fixed in https://github.com/creativecommons/cc-link-checker/pull/13/commits/0f48192515e4cc6554d37184b26401aaaeb9b8d7